images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -4,12 +4,91 @@ Describe "7-Zip" {
     }
 }
 
+Describe "AliyunCli" {
+    It "AliyunCli" {
+        "aliyun version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "AWS" {
+    It "AWS CLI" {
+        "aws --version" | Should -ReturnZeroExitCode
+    }
+
+    It "Session Manager Plugin for the AWS CLI" {
+        session-manager-plugin | Out-String | Should -Match "plugin was installed successfully"
+    }
+
+    It "AWS SAM CLI" {
+        "sam --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "AzCopy" {
```

Comment:
may be create a global `Describe "Azure"` and move AzCopy, Azure Cosmos DB Emulator, Azure DevOps CLI tests to it to combine tools by the author?

Reply:
These are quite different tools, the only similarity is Azure in the naming